src/main/java/baseball/model/Result.java

```diff
@@ -0,0 +1,42 @@
+package baseball.model;
+
+public class Result {
+    private static final int GAME_FINISH_COUNT = 3;
+    private static final int LIMIT_COUNT = 0;
+    private final int strike;
+    private final int ball;
+
+    public Result(int strike, int ball) {
+        this.strike = strike;
+        this.ball = ball;
+    }
+
+    public boolean isGameFinish() {
+        if (strike == GAME_FINISH_COUNT) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return strike == GAME_FINISH_COUNT;
```
다음과 같이 인라인으로 작성해주시면 코드가 조금 더 깔끔해질 수도 있을 것 같습니다